src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+  </head>
+  <body>
+    <section class="wrapper">
+      <header>
+        <h1>Leaderboard</h1>
+      </header>
+      <div class="board">
+          <div  class="user_board">
+            <label for="btn">Recent scores</label><button id="btn" type="button" name="button">Refresh</button>
+            <ul class="list_style list">
+              <li>No New Players</li>
+            </ul>
+          </div>
+
+          <form class="input_data">
+            <h2>Add your score</h2>
+            <input id="name" type="text" name="name" placeholder="Your name">
+            <input id="score" type="text" name="score" placeholder="Your score">
```

Comment:
OPTIONAL
- [ ] It will be good if you change the type of the score to the number. Because ı can add text as a score on the page.

![image](https://user-images.githubusercontent.com/38635158/175387662-88b0df91-b4e1-4367-9559-366343c74884.png)